source/developers/how-tos/enable-translations-new-repo.rst

```diff
@@ -93,7 +86,25 @@ edX project. There are many Django microservices in the Open edX ecosystem, such
 #. If you are creating a new repository, use the `cookiecutter-django-ida`_
    template.
 
-**TODO:** How to enable translations infrastructure on a Django microservice
+   For existing repos which don't have the ``make extract_translations`` or
+   ``make pull_translations`` command, they can be copied from the
+   `edx-cookiecutters`_ Makefile in the ``cookiecutter-django-ida`` Makefile for Microservices.
```

Comment:
It seems a little odd to have this list item start with "If you are creating a new repository," and then have "For existing repos" under the same list item.

Maybe having the "headline" of the list item be:
* Ensure your repository's makefile has `extract_translations` and `pull_translations`

and then having the instructions for new/existing repos under that would help clear it up